python/python/tests/test_async_client.py

```diff
@@ -3320,6 +3320,25 @@ async def test_pfadd(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.pfadd("foo", [])
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_object_freq(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        maxmemory_policy_key = "maxmemory-policy"
+        config = await redis_client.config_get([maxmemory_policy_key])
+        maxmemory_policy = cast(str, config.get(maxmemory_policy_key))
+
+        try:
+            assert (
+                await redis_client.config_set({maxmemory_policy_key: "allkeys-lfu"})
+                == OK
+            )
+            assert await redis_client.object_freq("non_existing_key") is None
```

Comment:
I think the chance that `non_existing_key` exists is higher than chance that `key` exists
```suggestion
            assert await redis_client.object_freq(key) is None
```